src/hist.jl

```diff
@@ -259,34 +272,42 @@ end
 append!{T,N}(h::AbstractHistogram{T,N}, vs::NTuple{N,AbstractVector}, wv::WeightVec) = append!(h, vs, values(wv))
 
 
+# Turn kwargs nbins into a type-stable tuple of integers:
+function _nbins_tuple{N}(vs::NTuple{N,AbstractVector}, nbins)
```

Comment:
Agreed - which version should I use, though? I have a `v"0.6.0-pre.alpha.206` here, and it supports broadcast for tuples. But if it should be `v"0.6.0+XXXX"` or `v"0.6.0-dev.XXXX"`, I'm not sure which number to use.